packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -664,6 +666,7 @@ export class Combobox extends FormAssociatedCombobox {
     private updateValue(shouldEmit?: boolean) {
         if (this.$fastController.isConnected) {
             this.value = this.firstSelectedOption?.text || this.control.value;
+            this.control.value = this.value;
```

Comment:
@radium-v,  I recognized another bug while testing changes. If `this.value` gets updated to the first option (`firstSelectedOption.text`), then `this.control.value` will not represent the current value, which is a problem. I've added a test to reflect this change.

Reply:
Thinking about this a bit more, I'm not sure why `this.control.value` doesn't automatically get updated when `this.value` is set through the template binding. Thoughts @radium-v?

Reply:
@EisenbergEffect is it possible that this is a bug? `this.control` relates to a ref in the template, and `this.value` is observable using the binding syntax as such:

```ts
html`
    <template>
        <input :value="${x => x.value}" ${ref("control")}>
    </template>
`
```

Reply:
Writing to the DOM happens on the update queue. So, if a binding updates value, you have to wait for the queue to be processed before the DOM is updated. Writing to models is synchronous but writing to the DOM is "eventually consistent". This is for performance reasons. That would be my guess at what's going on here based on the quick read of the above.

Reply:
I'll assume that I can just leave this change in place. (as I'm not sure what the alternative to correcting the bad behavior would be).